server/graphql/resolvers/auth.js

```diff
@@ -1,35 +1,34 @@
-const bcrypt = require('bcryptjs')
-const jwt = require('jsonwebtoken')
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
 
-const User = require('../../models/user')
+const User = require('../../models/user');
 const secret = process.env.SECRET || 'testingJWT';
 
 module.exports = {
-  login: async ({
-    email,
-    password
-  }) => {
+  login: async ({ email, password }) => {
     const user = await User.findOne({
       email: email
     });
     if (!user) {
       throw new Error('username or password is incorrect');
     }
-    const isAuthenticated = await bcrypt.compare(password, user.password)
+    const isAuthenticated = await bcrypt.compare(password, user.password);
```

Comment:
This might already be taken care of in a different module, but wondering if it possible to salt and hash the user's password?